patches/extensions-common-api-_permission_features.json.patch

```diff
@@ -1,28 +1,20 @@
 diff --git a/extensions/common/api/_permission_features.json b/extensions/common/api/_permission_features.json
-index 8ea8fa8b7a675cf04e8444b2a9daff40249cbdd4..cdbb8bd808aeaaa38712c4a4be06e6034baf8325 100644
+index f69bdd43d7c2f2be54df55514dff687fb6b63321..d6a88a53f9c87b9b70d9d2f811a15e4f74c99685 100644
 --- a/extensions/common/api/_permission_features.json
 +++ b/extensions/common/api/_permission_features.json
 @@ -314,6 +314,7 @@
        "channel": "stable",
        "extension_types": ["extension"],
        "allowlist": [
 +        "21070F3D60711361C1210B870439BE49B5D995F4",  // Ethereum Remote Client extension
```

Comment:
fyi - https://github.com/brave/brave-core/pull/9061/files#r1046314436

Reply:
Moved to our override in 26abd06a0409fc18aecebe4ae305361d9c59c733. Will create a follow up issue for using chromium_src override.

Reply:
Follow up issue: https://github.com/brave/brave-browser/issues/27314